liquidity/autoloop_test.go

```diff
@@ -146,6 +146,7 @@ func TestAutoLoopEnabled(t *testing.T) {
 			SweepConfTarget: params.SweepConfTarget,
 			OutgoingChanSet: loopdb.ChannelSet{chanID1.ToUint64()},
 			Label:           labels.AutoOutLabel(),
+			Initiator:       autoloopSwapInitiator,
```

Comment:
Really like being able to identify swaps from the autolooper! Wondering what the plan will be if we add autoloop to the LiT UI? Ideally we'd be able to easily update this to `LiT-autoloop`, will that be easy enough?

Reply:
Good point, I haven't thought about that yet. My suggestion is to add it to the liquidity parameters of the autolooper. Then the UI can overwrite the default value similar to how it's done for normal swaps.